src/resources/common/enums.ts

```diff
@@ -110,6 +110,7 @@ export enum QRGateway {
     ALIPAY_PLUS = "alipay_plus",
     AU_PAY = "au_pay",
     BARTONG = "bartong",
+    HELLOMONEY = "hellomoney",
```

Comment:
should this be
```
HELLO_MONEY = "hello_money",
```
?

Reply:
No. As per the link on the slack thread: https://github.com/univapaycast/gyron-payments-api/pull/6000, this is `hellomoney`. Note: this is an API key, we can not change it on frontend.

Reply:
oh I see thanx! resolving!